mantidproject/mantid
# MultivariateGaussianComptonProfile

Adds a fit function for fitting a multivariate Gaussian Compton profile.

Note that the `Sigma[XYZ]` parameters set the initial values in the `MultivariateGaussianMassProfile` and do not fix the values like other profiles. This is noted in code but should be made obvious in any documentation that is created.

**To test:**

Run this [script](https://gist.github.com/DanNixon/738080728276b2cfdb6c28c2a2a1d537), see that the fit is good and the parameter values and errors are reasonable.

Fixes #10852.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
